src/figuresClasses.ts

```diff
@@ -1,19 +1,100 @@
 export interface Figure {
-
+  a?: number,
+  b?: number,
+  c?: number,
+  radius?: number,
+  width?: number,
+  height?: number,
+  getArea(): number,
+  color: 'red' | 'green' | 'blue',
```

Comment:
so here you'll use like this
```suggestion
  shape: Shape,
  color: Color,
  getArea(): number;
```